server/public/model/client4.go

```diff
@@ -733,7 +742,7 @@ func (c *Client4) DoAPIDeleteJSON(ctx context.Context, url string, data any) (*h
 // DoAPIRequestWithHeaders makes an HTTP request with the specified method, URL, and custom headers.
 // Returns the HTTP response or any error that occurred during the request.
 func (c *Client4) DoAPIRequestWithHeaders(ctx context.Context, method, url, data string, headers map[string]string) (*http.Response, error) {
-	return c.doAPIRequestReader(ctx, method, url, "", strings.NewReader(data), headers)
+	return c.doAPIRequestReader(ctx, method, c.APIURL+url, "", strings.NewReader(data), headers)
```

Comment:
Always nervous when I see `+` joining URL components -- should we try to find a way to use `JoinPath` here instead?